src/Avalonia.Base/Input/PointerPoint.cs

```diff
@@ -155,17 +160,18 @@ public PointerPointProperties(RawInputModifiers modifiers, PointerUpdateKind kin
         }
 
         public PointerPointProperties(RawInputModifiers modifiers, PointerUpdateKind kind,
-            float twist, float pressure, float xTilt, float yTilt
+            float twist, float pressure, float xTilt, float yTilt, Rect contactRect
```

Comment:
Adding a new parameter breaks public API. You can either add a second overload, or use `public Rect ContactRect { get; init; }`

Reply:
Address.